src/checker.rs

```diff
@@ -212,7 +212,17 @@ impl Checker for TypescriptCode {
         ClassDeclaration,
         ArrowFunction
     );
-    mk_else_if!(IfStatement);
```

Comment:
It seems for both Rust and TypeScript you made the same change. What if we changed the `mk_else_if!` macro instead? Is there any language which still needs the previous implementation?

Reply:
JavaScript, Python,  Mozcpp needs the previous implementation